src/probnum/randvars/_arithmetic.py

```diff
@@ -319,10 +327,23 @@ def _matmul_constant_normal(constant_rv: _Constant, norm_rv: _Normal) -> _Normal
             cov=constant_rv.support @ (norm_rv.cov @ constant_rv.support.T),
             cov_cholesky=cov_cholesky,
         )
+    elif norm_rv.ndim == 2 and norm_rv.shape[0] > 1:
+        # This part does not do the Cholesky update,
+        # because of performance configurations: currently, there is no way of switching
+        # the Cholesky updates off, which might affect (large, potentially sparse) covariance matrices
+        # of matrix-variate Normal RVs. See Issue #335.
+        cov_update = _linear_operators.Kronecker(
+            constant_rv.support,
+            _linear_operators.Identity(norm_rv.shape[1]),
+        )
```

Comment:
I believe that this does not allow for vector @ matrix, where vector.ndim == 1.
However, numpy behaves correctly in this situation.

```suggestion
    elif norm_rv.ndim == 2 and norm_rv.shape[0] > 1:
        if constant_rv.support.ndim == 1:
            constant = constant_rv.support[:, None]
        else:
            constant = constant_rv.support

        # This part does not do the Cholesky update,
        # because of performance configurations: currently, there is no way of switching
        # the Cholesky updates off, which might affect (large, potentially sparse) covariance matrices
        # of matrix-variate Normal RVs. See Issue #335.
        cov_update = _linear_operators.Kronecker(
            constant,
            _linear_operators.Identity(norm_rv.shape[1]),
        )
```

- [ ] if we change this, the tests should probably be adjusted, too